docs/user_guide/build_hdl.rst

```diff
@@ -514,6 +517,99 @@ Starting with Vivado 2019.3, the output file extension was changed from
    script will create a board design in IPI (IP Integrator), generate all the
    IP targets, synthesize the netlist and implementation.
 
+Building a Lattice project
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The Lattice build is in a very early version, does not support any ADI library
+builds yet. We are just starting to develop the library build part.
+Currently we only have a single early version base design that builds almost
+like the same then the others. For Lattice there are separate tools for creating
+a block design **(Propel Builder)** and building a hdl design **(Radiant)**.
+Like in the other cases the build for any supported project works by ``make``.
+First you have to open the **Propel Builder GUI** and **download** the necessary
```

Comment:
`IP on server` only exists inside Propel Builder with a project open? There is no cli to install it by id (e.g `latticesemi.com:ip:i2c_controller:2.0.0`)?
Perhaps:
* mention that you launch it with `./launch_propel.sh`.

Reply:
You can download IPs from server in Propel Builder GUI, no project needs to be open. The IPs download folder in default is the home/PropelIPLocal. If an IP is downloaded, you should find it there. The content of that folder is shown in GUI at 'IP on Local' list.
![image](https://github.com/analogdevicesinc/hdl/assets/138444259/63f8db1d-6b15-4ce5-bea6-d328f3aaaaf1)

As far as I know there is only the GUI option for downloading IPs. I did not find any command line option in the documentation. 
Other than that, I think I have to doublecheck the available IP versions on the server, because since I created this pull request some IPs appeared with new name starting with version 2.0.0 and the latest versions with the old names like 1.x.x may be unavailable on the server.